.azure-pipelines/run_docker_build.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
```

Comment:
This file is duplicated?

Reply:
yes.  It comes from the same template.  In the case where circle building is disabled we still want it to be generated.

Reply:
Also this particular one is not exactly the same as the azure stuff has publishing artifacts disabled for now.